### DISCUSSION TITLE
user permissions

### DISCUSSION THREAD

**Discussion by lovestudylalala** [apache/apisix#5605] [Q&A]
How to divide apisix-dashboard user login permissions

**Comment by moonming** [comment]
@juzhiyuan please take a look

**Comment by juzhiyuan** [comment]
Hi @lovestudylalala, just to make sure, do you mean how to implement the User and Permission Management System in the Dashboard? e.g RBAC.

Here're 2 cases IMO,

1. Only need simple permission management, this needs to add some properties here[1], and write some Golang codes to check user permissions;
2. You need a complete RBAC system, for now, it doesn't support that. 🤔 If you really need it, I'd like to work with you to implement this **big feature**.

No matter which case, we all need to do some logic changes, that's because the current apisix-dashboard's destination is to **let users operate Apache APISIX smoothly**, so we don't support many features. Maybe in the next version (V3), we could support more **Scenario-based** features.

[1] https://github.com/apache/apisix-dashboard/blob/master/api/conf/conf.yaml#L70-L74

**Reply by lovestudylalala** [reply]
thanks
